src/sentry/static/sentry/app/components/events/contextSummary/filterContexts.tsx

```diff
@@ -1,10 +1,14 @@
-function filterContexts(event, context) {
+import {Event} from 'app/types';
+
+import {Context} from './contextSummary';
+
+function filterContexts(event: Event, context: Context) {
   // if the operating system is macOS, we want to hide devices called "Mac" which don't have any additional info
   if (context.keys.includes('device')) {
-    const {model, arch, data_id} = event.contexts?.device || {};
+    const {model, arch} = event.contexts?.device || {};
```

Comment:
are you sure about removing "data_id"? wasn't it being used?

Reply:
I looked in `snuba/events` and test pysnaps but couldn't find this field anywhere